site/en/blog/enabling-shared-array-buffer/index.md

```diff
@@ -7,11 +7,11 @@ description: >
   SharedArrayBuffer will arrive in Android Chrome 88. It will only be available
   to pages that are cross-origin isolated. Starting in Desktop Chrome 91 it will
   also only be available to cross-origin isolated pages. You can register for an
-  origin trial to retain the current behavior until Desktop Chrome 93.
+  origin trial to retain the current behavior until Desktop Chrome 96.
```

Comment:
These changes all seem unrelated. I don't mind them going in but is there a reason they're not in another PR?

Reply:
This may have been a mistake on my part. I wanted to get Solomon's branch up to date, so I merged `main` into his PR branch. That picked up a bunch of unrelated changes. I think I may need to create a new branch off of `main` and cherry pick his commits in order to undo what I did.